etc/ReplayOfflineConfiguration.py

```diff
@@ -401,7 +402,7 @@
 #####################
 
 addExpressConfig(tier0Config, "HIExpress",
-                 scenario=hiTestppScenario,
+                 scenario=HIalcaTrackingOnlyScenario,
```

Comment:
this should be `hiTestppScenario` no?

Reply:
Ok I see you have changed the `HIExpress` scenario instead of the `HIExpressAlignment`

Reply:
Ohh yeah, my bad. Fixing.